### ISSUE TITLE
[css-text-4] Text-spacing is too strict

### ISSUE THREAD

**Issue by himorin** [w3c/i18n-activity#742]
**This is a tracker issue.** Only discuss things here if they are i18n WG internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ https://github.com/w3c/csswg-drafts/issues/3229
Add flexibility on inserting spacing between "ideographs" and "non-ideographic letters".

**Comment by xfq** [comment]
A new `text-spacing: auto` value, which is not the initial value, has been added: https://github.com/w3c/csswg-drafts/pull/4657/files

**Comment by xfq** [comment]
Should we create GitHub issues to track other ways of adding spacing between ideographs and non-ideographic letters and rules of fullwidth punctuation collapsing in clreq/jlreq?

**Comment by himorin** [comment]
Resolved with adding auto to text-spacing as non-initial, by PR https://github.com/w3c/csswg-drafts/pull/4657

**Comment by himorin** [comment]
> Should we create GitHub issues to track other ways of adding spacing between ideographs and non-ideographic letters and rules of fullwidth punctuation collapsing in clreq/jlreq?

I suppose adding extra spacing with ideograph-* values could be simple, since it just adds almost without relationship with typography or font, but reducing spacing is much complex as pointed.
I agree with having another issue(s) on these point, but I feel we'd also be better to have discussion within (at least) JLReq for that. In parallel, as @murata2makoto pointed, we should have clear idea on how we can have better coordination with font, like non-turing glyphs in vertical writing mode.

**Comment by xfq** [comment]
Related jlreq discussion on adding spacing between ideographs and non-ideographic letters: https://github.com/w3c/jlreq/issues/163

**Comment by aphillips** [comment]
there is discussion above after we marked close. Please reopen or file a new issue to replace this if needed @xfq @himorin
